<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsultWindow</class>
 <widget class="QMainWindow" name="ConsultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Консультация</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="dialogFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="answerButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ответить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="answerCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="explainButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать объяснение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newConsultButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Новая косультация</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
